src/modules/drawing/components/DrawingToolbox.vue

```diff
@@ -206,6 +209,21 @@ $zindex-drawing-toolbox: -1;
         display: none;
     }
 }
+.drawing-button {
+    touch-action: manipulation;
+    cursor: pointer;
+    background-image: none;
+    background-color: transparent;
+    border: 1px solid transparent;
+    border-radius: 4px;
+    width: 33%;
+}
```

Comment:
Lots of the properties in the "drawing-button" class (like background-image and background-color) seem to be the default values. Maybe you could remove the attributes that are not necessary.

Reply:
I removed what was necessary to remove. I left the background-color as it is not transparent by default and we need it to be transparent.